tests/benchmark/broker-benchmark.cc

```diff
@@ -242,9 +241,30 @@ void client_mode(endpoint& ep, const std::string& host, int port) {
   }
   if (verbose)
     std::cout << "*** endpoint is now peering to remote" << std::endl;
+  if (batch_rate == 0) {
+    ep.publish_all( //
+      [](caf::unit_t&) {},
+      [](caf::unit_t&, caf::downstream<data_message>& out, size_t hint) {
+      for (size_t i = 0; i < hint; ++i) {
+      auto name = "event_" + std::to_string(event_type);
+      out.push(data_message{"/benchmark/events",
+               zeek::Event(std::move(name), createEventArgs())});
+      }
+      },
+      [](const caf::unit_t&) { return false; }
+      );
+    for (;;) {
+      // Print status events.
+      auto status_events = ss.poll();
+      if (verbose)
+        for (auto& ev : status_events)
+          std::cout << caf::deep_to_string(ev) << std::endl;
+    }
```

Comment:
The `ss.poll()` call is non-blocking so this infinite loop spun as fast as it could doing nothing.  Didn't think there's a reason for that so I changed it to `ss.get()` in the merge commit, but let me know if the original way was intentional.

Reply:
No, that wasn't intentional. I've assumed `poll()` to block for some reason. Thanks for fixing!